java/ql/src/Security/CWE/CWE-927/SensitiveResultReceiver.ql

```diff
@@ -0,0 +1,21 @@
+/**
+ * @name Leaking sensitive information through a ResultReceiver
+ * @description Sending sensitive data to a 'ResultReceiver' from an untrusted source
```

Comment:
```suggestion
 * @description Sending sensitive data to a 'ResultReceiver' obtained from an untrusted source
```

Small clarification, please @sabrowning1 let me know if this sounds weird. I wanted to clarify that what comes from an untrusted source is the `ResultReceiver`, not the sensitive data.

Reply:
@atorralba that clarification makes sense to me! 👍🏼